com.eminfedar.vaktisalah-gtk.json

```diff
@@ -0,0 +1,101 @@
+{
+    "id": "com.eminfedar.vaktisalah-gtk",
```

Comment:
there is nothing found at `eminfedar.com`.

Reply:
Hmm, I think I can change it to "io.github.eminfedar.vaktisalah-gtk" but it still will be a non existing website. This program only needs a github repository, not a separate website.

Reply:
`io.github.eminfedar.vaktisalah-gtk` would be preferred even if there's no website at that address.

Reply:
Never mind, just noticed it's already done.